File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        long max = 0;
        long min = 0;
        
        max = Math.max(a, b);
        min = Math.min(a, b);
        
        for (long i = min; i <= max; i++) {
            answer += i;
        }
        
        return answer;
    }
}

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.cs
using System;

public class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        
        long A = Math.Min(a, b);
        long B = Math.Max(a, b);
        
        answer = (A + B) * (B - A + 1) / 2;
        
        return answer;
    }
}